<ui version="4.0" >
 <class>UIBuildStep</class>
 <widget class="pDockWidget" name="UIBuildStep" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowIcon" >
   <iconset resource="../resources/MessageBox.qrc" >:/icons/tabbuild.png</iconset>
  </property>
  <property name="windowTitle" >
   <string>Build Steps</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" >
    <item>
     <widget class="QListWidget" name="lwBuildSteps" />
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pDockWidget</class>
   <extends>QDockWidget</extends>
   <header location="global" >fresh.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/MessageBox.qrc" />
 </resources>
 <connections/>
</ui>
